samples/suit/recovery/README.rst

```diff
@@ -93,12 +93,22 @@ Testing
 |test_sample|
 
 #. |connect_kit|
-#. Corrupt the currently running main application (for example by flashing a modified version of the application)
-#. Open the Device Manager or the nRF Connect application, and observe the device advertising as "SUIT Recovery"
-#. Recover the application using `nRF Connect Device Manager`_ in the same way as described in the :ref:`nrf54h_suit_sample` sample documentation.
+#. Corrupt the currently running main application. This can be done by overwriting the beggining of the application partition.
```

Comment:
```suggestion
#. Corrupt the currently running main application.
   This can be done by overwriting the beginning of the application partition:
```